### DISCUSSION TITLE
What is the difference between Appdata / Program Files?

### DISCUSSION THREAD

**Discussion by laborantine** [pdfarranger/pdfarranger#745] [Q&A] [answered]
Hello and sorry for my English, I am French.

First of all, I want to congratulate the teams working on this application because it is very practical for the management of .pdf.

I am part of an IT department, equipped with a computer park where this type of software can interest several hundred people.

I would like to understand why I chose to install this application in the user's "Appdata" folder rather than "Program Files".

As part of a test, we managed to silently install the program using msiexec by specifying the command "TARGETDIR=C:\Program Files\PDF Arranger". But I think it can cause problems, especially when uninstalling the product.

Can you tell me if it is possible to deploy this application in "Program Files" and the repercussions this could have?

Have a good day !

**Comment by jeromerobert** [comment, marked as answer]
Installing to the `Program Files` folder should work properly. If you find any issue please report here.

I choose to default to `AppData` because of the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege).

**Comment by laborantine** [comment]
Hello,

I come back to you regarding the problem I encountered.

Everything is fine now on 1.9.1.

At the time when I was deploying the version 1.8.2, the "TARGETDIR" only caused problems and seemed not to complete the installation completely (missing shortcut, error during launch...etc).

By putting this line: " msiexec /i "pdfarranger-1.9.1-windows-installer.msi" ALLUSERS=1 INSTALLDIR="C:\Program Files\PDF Arranger" TARGETDIR="C:\Program Files\PDF Arranger" " it works correctly.

I didn't realize at the time, maybe I downloaded a corrupt msi or whatever.

Our users are generally satisfied with this app and we look forward to future updates. Thank you for the explanations on Principle of least privilege, it's interesting to know the risks and the advantages. 👍
